roles/molecule/Dockerfile.j2

```diff
@@ -0,0 +1,22 @@
+# Forked from https://github.com/ansible-community/molecule-docker/blob/main/src/molecule_docker/playbooks/Dockerfile.j2
```

Comment:
having this folder suggests we have a `molecule` role which is not true. I'd rather we use the root `/molecule` folder for that